packages/vitest/src/node/types/coverage.ts

```diff
@@ -71,7 +78,7 @@ export interface CoverageProviderModule {
   /**
    * Executed on after each run in the worker thread. Possible to return a payload passed to the provider
    */
-  takeCoverage?: () => unknown | Promise<unknown>
+  takeCoverage?: (runtimeOptions: { moduleExecutionInfo?: ModuleExecutionInfo }) => unknown | Promise<unknown>
```

Comment:
If you want to keep this backwards compatible, it should be this:

```suggestion
  takeCoverage?: (runtimeOptions?: { moduleExecutionInfo?: ModuleExecutionInfo }) => unknown | Promise<unknown>
```